core/src/main/java/io/grpc/internal/InsightBuilder.java

```diff
@@ -25,7 +26,7 @@
  * information.
  */
 public final class InsightBuilder {
-  private final ArrayList<String> buffer = new ArrayList<String>();
+  private final List<String> buffer = new ArrayList<>();
```

Comment:
`ArrayList` is fine (just remove `<String>`). See https://github.com/grpc/grpc-java/pull/5892#discussion_r295381892

Reply:
okay.